test/Graphics/Shape.cpp

```diff
@@ -0,0 +1,89 @@
+#include <SFML/Graphics/Shape.hpp>
+#include "GraphicsUtil.hpp"
+
+#include <doctest.h>
+
+using doctest::Approx;
+
+class TriangleShape : public sf::Shape
+{
+public:
+    explicit TriangleShape(const sf::Vector2f& size);
+    std::size_t getPointCount() const override;
+    sf::Vector2f getPoint(std::size_t index) const override;
+private:
+    sf::Vector2f m_size;
+};
+
+TEST_CASE("sf::Shape class - [graphics]")
+{
+    SUBCASE("Default constructor")
+    {
+        const TriangleShape triangleShape({0, 0});
+        CHECK(triangleShape.getTexture() == nullptr);
+        CHECK(triangleShape.getTextureRect() == sf::IntRect());
+        CHECK(triangleShape.getFillColor() == sf::Color::White);
+        CHECK(triangleShape.getOutlineColor() == sf::Color::White);
+        CHECK(triangleShape.getOutlineThickness() == 0.0f);
+        CHECK(triangleShape.getLocalBounds() == sf::FloatRect());
+        CHECK(triangleShape.getGlobalBounds() == sf::FloatRect());
+    }
+
+    SUBCASE("Setters and getters")
```

Comment:
Personally, I'd prefer one case per functionality instead of just throwing everything into the mix 😃

Reply:
Do you want one test case per setter/getter combo?

Reply:
I'd do it this way. So if anything breaks, I immediately know it's in the XYZ setter/getter and unrelated to the UVW setter/getter